.gitignore

```diff
@@ -66,3 +66,4 @@ target/
 build/
 
 yarn.lock
+.idea
```

Comment:
**question**: Why does it ignore the `.idea` file? What is this file? Could we remove it if it is not needed?

Reply:
This folder contains the configuration files of the WebStorm IDE, it is similar to the .vscode folder.